proposals/dao/tip_118.ts

```diff
@@ -0,0 +1,392 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  PcvStats,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner, overwriteChainlinkAggregator, time } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+import { BigNumber } from 'ethers';
+import { ERC20HoldingPCVDeposit } from '@custom-types/contracts';
+
+/*
+
+Tribal Council proposal TIP_118
+
+- Deploy ERC20Holding PCV Deposits for remaining PCV assets
+- Deprecate LUSD, RAI and ETH PSMs
+- Pause PCV Drip Controllers and deprecate skimmers
+- Complete deprecation of TRIBE incentives system by withdrawing excess TRIBE
+
+*/
+
+const toBN = BigNumber.from;
+
+const fipNumber = 'tip_118';
+
+let pcvStatsBefore: PcvStats;
+let daiBalanceBefore: BigNumber;
+let initialCoreTribeBalance: BigNumber;
+let initialRariDelegatorBalance: BigNumber;
+
+const EXPECTED_WETH_TRANSFER = toBN('21716293570455965978548');
```

Comment:
Please add a comment for each of these vars explaining what they mean & providing a little bit of context